src/store/modules/PlantumlEditor.js

```diff
@@ -15,8 +15,8 @@ const state: any = {
   plantuml: 'plantuml',
   server: process.env.VUE_APP_SERVER,
   cdn: process.env.VUE_APP_CDN,
-  startuml: ['@startuml', '@startmindmap', '@startditaa', '@startgantt', '@startwbs'],
-  enduml: ['@enduml', '@endmindmap', '@endditaa', '@endgantt', '@endwbs'],
+  startuml: ['@startuml', '@startmindmap', '@startditaa', '@startgantt', '@startwbs', '@startjson', 'startsalt'],
```

Comment:
I think `startsalt` also needs a `@` in the beginning.
In this case it would be `'@startsalt'`

Reply:
`@startyaml`, `@startmath` and `@startlatex` and and their respective end token are also missing.

References:
- [yaml](https://plantuml.com/yaml)
- [math and latex](https://plantuml.com/ascii-math)

As an alternative to manually maintaining all start and end tokens, one might also consider accepting everything with `@start` and `@end`.